tests/test.ts

```diff
@@ -43,6 +43,31 @@ describe.skip('Test clasp create function', () => {
   });
 });
 
+// When running this test, please put in a script ID
+// then use `sudo npm run build` before running test.
```

Comment:
Don't you need to build before any test?

Shouldn't we have this in the file header comment if that is the case?

Reply:
Yeah good point